Sources/Common/DataModel/ImageData/index.js

```diff
@@ -233,6 +234,21 @@ function vtkImageData(publicAPI, model) {
     return bounds;
   };
 
+  // Get array extents as float / spatial coordinates
+  // without scaling for voxel/pixel spacing.
+  publicAPI.getSpatialExtent = () => {
+    const ext = model.extent;
+    const spatialExt = [
+      ext[0] - 0.5,
+      ext[1] + 0.5,
+      ext[2] - 0.5,
+      ext[3] + 0.5,
+      ext[4] - 0.5,
+      ext[5] + 0.5,
+    ];
+    return spatialExt;
```

Comment:
could have been `return vtkBoundingBox.inflate(...model.extent, 0.5)` (inflate() would need to return something though...

Reply:
Made appropriate changes.